lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1724,4 +1747,270 @@ bool PrimUpdaterManager::readPullInformation(const MDagPath& dagPath, Ufe::Path&
     return false;
 }
 
+/* static */
+std::list<PrimUpdaterManager::VariantSetDescriptor>
+PrimUpdaterManager::variantSetDescriptors(const Ufe::Path& p)
+{
+    std::list<VariantSetDescriptor> vsd;
+    auto                            path = p;
+    while (path.nbSegments() > 1) {
```

Comment:
Maybe the loop condition could be that the run-time ID is still mayaUsd? Could be more future-proof.